src/app/docs/radio/radio-docs.component.html

```diff
@@ -1,13 +1,333 @@
 <sky-docs-demo-page
   moduleName="SkyRadioModule"
   moduleSourceCodePath="src/app/public/modules/radio/"
-  pageTitle="Radio"
+  pageTitle="Radio button"
 >
   <sky-docs-demo-page-summary>
-    Sample description.
+    The radio button module ceates a small set of selection inputs and allows users to select one option only.
```

Comment:
It's superminor, but I'd change `and allows` to `for` and delete `only`. Our phase 2 overview descriptions have tried to get away from implementation details and succinctly describe what the components do or create with an active explanation of what the component does rather than what it allows someone to do.

Reply:
✔️